pyfar/dsp/classes.py

```diff
@@ -184,6 +187,16 @@ def reset(self):
             warnings.warn(
                 "No previous state was set. Initialize a filter state first.")
 
+    @property
+    def comment(self):
+        """Get comment."""
+        return self._comment
+
+    @comment.setter
+    def comment(self, value):
+        """Set comment."""
+        self._comment = value
```

Comment:
I think it makes sense here to store the input as a string in the setter in case of numerical input.

Reply:
you mean `self._comment = str(value)` ?

Reply:
Yeah, in case the user inputs something like `self._comment = str(500)`